src/client/pages/RetroCertsConfirmationPage/__snapshots__/index.test.js.snap

```diff
@@ -62,7 +62,7 @@ exports[`<RetroCertsConfirmationPage /> with confirmation number 1`] = `
           t={[Function]}
           values={
             Object {
-              "confirmationNumber": "CONFIRMATION_NUMBER",
+              "confirmationNumber": "_NUMBER",
```

Comment:
Interesting, I don't know why this snapshot changed in this way...

Reply:
Haha, in the test, it sets the confirmation number to CONFIRMATION_NUMBER. Your code is truncating to the last 7 digits :)